bin/azure-deploy

```diff
@@ -57,19 +67,13 @@ sed \
   -e "s/\${keyVaultName}/$KEYVAULT_NAME/g" \
   "$SECRETS_PARAMETERS_TEMPLATE_FILE_PATH" > "$SECRETS_PARAMETERS_FILE_PATH"
 
-echo
-echo "Are you ready to deploy secrets?"
-echo "  Hit return to continue, or CTRL+C to stop."
-read -r
-
 echo "Starting secrets deployment..."
 az group deployment create \
   --name "$SECRETS_DEPLOYMENT_NAME" \
   --resource-group "$SECRETS_RESOURCE_GROUP_NAME" \
   --template-file "$SECRETS_TEMPLATE_FILE_PATH" \
   --parameters "@$SECRETS_PARAMETERS_FILE_PATH" \
   --mode Complete \
-  --rollback-on-error \
```

Comment:
This should only ever be an issue on first deploy. I had considered using a flag on `bin/azure-deploy` to skip this flag on first deployment. Maybe now is the time to do that?